frontend/server/src/test/java/org/pytorch/serve/util/ConnectorTest.java

```diff
@@ -0,0 +1,117 @@
+package org.pytorch.serve.util;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ConnectorTest {
+
+    @Test
+    public void testValidManagementAddress() {
+        Connector conn =
+                Connector.parse("http://127.0.0.1:45245", ConnectorType.MANAGEMENT_CONNECTOR);
+
+        Assert.assertEquals(conn.getSocketType(), "tcp");
+        Assert.assertEquals(conn.getSocketPath(), "45245");
+        Assert.assertTrue(conn.isManagement());
+        Assert.assertFalse(conn.isSsl());
+        Assert.assertFalse(conn.isUds());
+    }
+
+    @Test
+    public void testValidInferenceAddress() {
+        Connector conn =
+                Connector.parse("http://127.0.0.1:45245", ConnectorType.INFERENCE_CONNECTOR);
+
+        Assert.assertEquals(conn.getSocketType(), "tcp");
+        Assert.assertEquals(conn.getSocketPath(), "45245");
+        Assert.assertFalse(conn.isManagement());
+        Assert.assertFalse(conn.isSsl());
+        Assert.assertFalse(conn.isUds());
+    }
+
+    @Test
+    public void testSSLInferenceAddress() {
+        Connector conn =
+                Connector.parse("https://127.0.0.1:45245", ConnectorType.INFERENCE_CONNECTOR);
+
+        Assert.assertEquals(conn.getSocketType(), "tcp");
+        Assert.assertEquals(conn.getSocketPath(), "45245");
+        Assert.assertFalse(conn.isManagement());
```

Comment:
@harshbafna Looks like copy/paste error in the Inference port check

Reply:
@chauhang: It's a negative assertion i.e. assertFalse